src/basic/cgroup-util.c

```diff
@@ -134,6 +134,11 @@ int cg_read_event(const char *controller, const char *path, const char *event,
         return -ENOENT;
 }
 
+bool cg_ns_supported(void)
+{
+	return access("/proc/self/ns/cgroup", F_OK) == 0;
```

Comment:
nitpick: please follow the usual coding style, and place the opening bracket on the same line as the function name. i.e.:

```c
bool cg_ns_supported(void) {
…
```